setup.cfg

```diff
@@ -54,6 +54,8 @@ addopts =
     --cov=daidepp
     --cov-report html
 timeout = 10
+filterwarnings =
```

Comment:
Just out of curiosity, what is this for?

Reply:
I was getting 
```
tests/test_grammar.py: 22 warnings
  <unknown>:1: DeprecationWarning: invalid escape sequence \d

tests/test_daide_visitor.py: 3 warnings
tests/test_grammar.py: 39 warnings
  <unknown>:1: DeprecationWarning: invalid escape sequence \s
 ```
  when I ran tests. I couldn't figure out how to get rid of it. If you figure out how to get rid of those, feel free to push the change to this branch.